tests/system/action/user/test_set_password_self.py

```diff
@@ -42,35 +41,41 @@ def test_set_password_self_anonymus(self) -> None:
         self.assert_status_code(response, 400)
         self.assertIn("Can't set password for anonymous", response.json["message"])
 
-    # TODO: fix when permission is correctly implemented
-    @pytest.mark.skip()
-    def test_set_password_self_temporary_permissions(self) -> None:
+    def test_set_password_self_permissions(self) -> None:
         self.create_meeting()
-        self.set_group_permissions(1, [Permissions.User.CAN_CHANGE_OWN_PASSWORD])
         self.user_id = self.create_user("test", group_ids=[1])
+
         self.login(self.user_id)
         old_hash = self.auth.hash("old")
-        self.update_model("user/2", {"password": old_hash, "meeting_id": 1})
+        self.update_model(
+            "user/2",
+            {"password": old_hash, "meeting_ids": [1], "can_change_own_password": True},
+        )
         response = self.request(
```

Comment:
This test is duplicated to the first test `test_update_correct_permission` due to the fact that it is not meeting dependend anymore.

Reply:
This is removed.